docs/references.bib

```diff
@@ -49,3 +49,21 @@ @ARTICLE{2019ApJS..241...22D
        adsurl = {https://ui.adsabs.harvard.edu/abs/2019ApJS..241...22D},
       adsnote = {Provided by the SAO/NASA Astrophysics Data System}
 }
+
+@ARTICLE{2004ApJ...609.1181P,
+       author = {{Przybilla}, Norbert and {Butler}, Keith},
+        title = "{Non-LTE Line Formation for Hydrogen Revisited}",
+      journal = {\apj},
```

Comment:
```suggestion
      journal = {Astrophysical Journal},
```
The documentation does not render the journal abbreviations.

Reply:
In principle, this also needs to be changed for the other entries in the bib file.